pkg/networkmanager/manifest.json.in

```diff
@@ -12,6 +12,9 @@
         }
     },
 
+    "keywords": ["networks", "firewall", "intefaces", "bridge", "vlan", "bond", "team", "ports", "mac", "ipv4", "ipv6",
```

Comment:
Typo: "intefaces" presumably should be "interfaces".

(which raises the question: what happens if the user misspells their search term?)

Reply:
thanks :)

> what happens if the user misspells their search term?

Not much. We would need to implement some fuzzy searching.

Reply:
Also, IMHO we should establish a convention to use singular forms. If someone searches for "interface" they should get a hit.